extensions/community_extensions/Protected Audience Support.md

```diff
@@ -31,6 +31,15 @@
     1 = yes, the buyer is allowed
     </td>
   </tr>
+<tr>
+    <td><code>pbsmap</code></td>
+    <td>integer;<br>default 0</td>
+    <td>Indicates whether the seller can provide <code>igb.pbs</code> in <code>Object: InterestGroupAuctionBuyerSignals</code></br>
+    where <br>
+	0 = no; igb.pbs value is passthrough (DEFAULT)<br>
```

Comment:
We should just choose the best place, not provide two different places to put the thing. We should only do 1, yes, below